test/playwright/grafana.test.ts

```diff
@@ -46,7 +46,7 @@ test("validate namespace dashboard", async ({ page }) => {
     await page.goto(`/dashboards`);
     await page.click('text="Kubernetes / Compute Resources / Namespace (Pods)"');
     await page.getByTestId('data-testid Dashboard template variables Variable Value DropDown value link text authservice').click();
-    await page.getByRole('option', { name: 'grafana' }).click();
+    await page.getByRole('option', { name: 'authservice-test-app' }).click();
```

Comment:
I think this may result in failure on the monitoring single-layer tests since the test apps are not deployed there (and these same tests are run for both). We should be able to conditionally set a different namespace here based on `fullCore` to cover each of the two tests (continuing to use `grafana` for `!fullCore`).